src/components/slidingPanel/js/SlidingPanel.js

```diff
@@ -112,4 +139,22 @@ var fluid_2_0 = fluid_2_0 || {};
         that.events[that.model.isShowing ? "onPanelShow" : "onPanelHide"].fire();
     };
 
+    // panelId is passed in to ensure that it is evaluated before this
+    // function is called.
+    fluid.slidingPanel.setAriaProperties = function (that, panelId) {
+        that.locate("toggleButton").attr({
```

Comment:
@amb26 I suspect that this is a case of habit rather than a thought out choice. It seems to me that as a general practice we should lean towards using "prop". The only reason I see to stick with "attr" is that it's all over the code base. What do you think? Should I change this use?